test/support/commands.js

```diff
@@ -28,7 +27,9 @@ Cypress.Commands.add('showPreview', (token, fileId, options) => {
 });
 
 Cypress.Commands.add('showControls', () => {
-    cy.getByTestId('bp').trigger('mouseover');
+    cy.getByTestId('bp-controls')
+        .invoke('show')
```

Comment:
Is the mouseover event no longer working for the controls?

Reply:
Yes, it seems that the `trigger` and `mouseover` doesn't bring up the controls panel anymore.

Reply:
Does that reflect an actual problem with the controls or does Cypress use a different mechanism to trigger mouseover events now? We want the end-to-end tests to act as much like a user as possible, so we try to avoid invoking functions directly from them.

Reply:
Hmm, no, doesn't seem like a problem with the controls; I think something changed with the trigger command - I see that if you don't pass it an option of `force: true` it [waits for actionability](https://docs.cypress.io/api/commands/trigger#Assertions). Perhaps that has something to do with it...I'll give it a try

Reply:
Huh, I was curious what happened on the old version of cypress with the old version of pdfjs, and it looks like the hover never happens but the test passes - i.e. `Thumbnails.e2e.test.js`.

Reply:
Per slack discussion, `mousemove` works!